packages/@o3r/apis-manager/package.json

```diff
@@ -68,6 +68,9 @@
     "typescript": "~4.9.4",
     "zone.js": "^0.12.0"
   },
+  "generatorDependencies": {
```

Comment:
should not be needed as part of the peerDeps

Reply:
removed